pom.xml

```diff
@@ -112,7 +112,6 @@
 	</ciManagement>
 
 	<properties>
-		<scijava.jvm.version>1.5</scijava.jvm.version>
```

Comment:
We cannot update this component to Java 8. It is used by Bio-Formats, which is still built on Java 7. And others in the developer community use it as well, some of whom might need it to remain at Java 5 or Java 6.